source/scripts/Provider/PaliProvider.ts

```diff
@@ -15,6 +17,51 @@ export const PaliProvider = () => {
 
   const getState = () => {};
 
+  const sendToken = async (items: any) => {
+    const controller = getController();
+
+    const handleSendToken = await controller?.connections?.handleSendToken(
+      items
+    );
+
+    return handleSendToken;
+  };
+
+  const createToken = async (items: any) => {
+    const controller = getController();
+
+    const handleCreateToken = await controller?.connections?.handleCreateToken(
+      items
+    );
+
+    return handleCreateToken;
+  };
+
+  const transferToken = async (items: any) => {
+    const controller = getController();
+
+    const handleTransferToken =
+      await controller?.connections?.handleTransferOwnership(items);
+
+    return handleTransferToken;
+  };
+
+  const signPSBT = async (psbtToSign: any) => {
+    const controller = getController();
+
+    const handleSignPSBT = await controller?.connections?.signPSBT(psbtToSign);
+
+    return handleSignPSBT;
+  };
+
+  const getSignedPSBT = async (psbtToSign: any) => {
+    const controller = getController();
+
+    return {
+      isSigned: Boolean(await controller?.connections?.signPSBT(psbtToSign)),
+    };
```

Comment:
```suggestion
    return controller?.connections?.signPSBT(psbtToSign);
```

signPSBT already returns the signed psbt object, so we don't need to check if it is signed